core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -1107,6 +1114,30 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
     }
   }
 
+  private void unParseInStr(SqlWriter writer, SqlCall call, int leftPrec, int rightPrec) {
+    final SqlWriter.Frame substringFrame = writer.startFunCall("INSTR");
+    for (SqlNode operand : call.getOperandList()) {
+      if (operand instanceof SqlCharStringLiteral && operand.toString().contains("\\")) {
+        operand = SqlUtil.replaceSingleBackslashes(operand);
+      }
+      writer.sep(",");
+      operand.unparse(writer, leftPrec, rightPrec);
+    }
+    writer.endFunCall(substringFrame);
```

Comment:
same as above : change this to instrFrame

Reply:
Done